reflex/components/core/layout/stack.py

```diff
@@ -0,0 +1,108 @@
+"""Stack components."""
+
+
+from typing import Literal
+
+from reflex.components.component import Component
+from reflex.components.el.elements.typography import Div
+from reflex.vars import Var
+
+
+class Stack(Div):
+    """A stack component."""
+
+    # The direction of the stack.
+    justify: Var[Literal["start", "center", "end"]]
+
+    # The alignment of the stack.
+    align: Var[Literal["start", "center", "end", "stretch"]]
+
+    # The spacing between each stack item.
+    spacing: Var[str]
+
+    @classmethod
+    def create(cls, *children, **props) -> Component:
+        """Create a new instance of the component.
+
+        Args:
+            *children: The children of the stack.
+            **props: The properties of the stack.
+
+        Returns:
+            The stack component.
+        """
+        style = props.setdefault("style", {})
+        align = props.pop("align", "center")
+        spacing = props.pop("spacing", "0.5rem")
```

Comment:
where does the 0.5rem come from? just what "feels nice"?

more broadly, extending these from `Div` and not radix `Flex` means we dont get the common layout and common margin props. I can see the benefits of both ways, but as it is now, if you want consistency with radix themes components, then you probably end up reimplementing these yourself anyway, but based on radix `Flex` or `Box`

Reply:
same default as chakra for the "0.5rem"